bpf/lib/common.h

```diff
@@ -742,4 +692,6 @@ static inline int redirect_peer(int ifindex, __u32 flags)
 # define EADDRINUSE		98
 #endif
 
+#include "overloadable.h"
```

Comment:
It's a bit unconventional to add a header `#include` at the end of the file, but I guess it's relying on something defined earlier here?

Reply:
Yeah, it depends on prior defined static inlines in this file.